auth-web/vue.config.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
       // swSrc is required in InjectManifest mode.
       swSrc: 'src/service-worker.js',
       // skip precaching json files such as configs
-      exclude: [/\.json$/]
+      exclude: [/\.json$/, /\.map$/, /^manifest.*\.js(?:on)?$/]
```

Comment:
![image](https://user-images.githubusercontent.com/3484109/177221093-dfac1978-ef64-466e-bfe6-a8666430142a.png)

Reply:
The Regex is a little bit different, I tested it in Regex calculator.